### PR TITLE
VZ-10984: Upgrade to Istio 1.19.0

### PR DESCRIPTION
**Pull request by prasad-shirodkar**
Upgrade to Istio 1.19.0
- istioctl binary will now be consumed as an image (instead of RPM package) when building the Verrazzano Platform Operator. This will help expedite uptake of newer versions of Istio in the future.

Additional fixes:
- fluentd component should depend on istio since fluentd is part of the istio service mesh.

### CODE REVIEW

**Review by anders-swanson — DISMISSED** [review]

**Review by mgianatagh — APPROVED** [review]

File: platform-operator/manifests/catalog/catalog.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ modules:
     valuesFiles:
       - platform-operator/helm_config/overrides/external-dns-values.yaml
   - name: istio
-    version: 1.17.2-1
+    version: 1.19.0-1
```
Comment (mbenmclean) [inline platform-operator/manifests/catalog/catalog.yaml:31]: remove the -1. So the version should just be 1.19.0

File: platform-operator/verrazzano-bom.json [hunk 2]
```diff
@@ -122,20 +122,20 @@
     },
     {
       "name": "istio",
-      "version": "1.17.2-1",
+      "version": "1.19.0-1",
```
Comment (mbenmclean) [inline platform-operator/verrazzano-bom.json:125]: 1.19.0

**Review by mbenmclean — CHANGES_REQUESTED** [review]

File: platform-operator/manifests/catalog/catalog.yaml (hunk 1 above)
Comment (prasad-shirodkar) [reply platform-operator/manifests/catalog/catalog.yaml]: Done

File: platform-operator/verrazzano-bom.json (hunk 2 above)
Comment (prasad-shirodkar) [reply platform-operator/verrazzano-bom.json]: Done

**Review by mgianatagh — APPROVED** [review]

**Review by mbenmclean — APPROVED** [review]

**Pull request merged**